Elevel/Application/Features/AudioFeatures/CreateAuditionCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Threading.Tasks;
+using MediatR;
+using Elevel.Application.Interfaces;
+using AutoMapper;
+using System.Threading;
+using Elevel.Domain.Models;
+
+namespace Elevel.Application.Features.AudioFeatures
+{
+    public class CreateAuditionCommand
+    {
+        public class Request : IRequest<Response>
+        {
+            public string AudioFilePath { get; set; }
+            public DateTimeOffset CreationDate { get; set; }
```

Comment:
Creation date should be set at your handler, clint should not send it.
and what path will be at audioFilePath?